packages/hash/api/src/model/util.ts

```diff
@@ -330,6 +330,9 @@ export type LinkEntityTypeCreatorParams = Omit<
   "entityTypeId"
 > & {
   linkEntityTypeId: VersionedUri;
+  title: string;
+  description?: string;
+  actorId: string;
```

Comment:
Do these not exist on `EntityTypeCreatorParams`?

Reply:
`description` actually wasn't, but have added as part of https://github.com/hashintel/hash/pull/1489/commits/ae331c6d485b3adc957d258418d4f012f70ac308